setup.py

```diff
@@ -71,7 +71,16 @@
         **extra
     )
 except Exception as e:
-    if isinstance(e.message, str) and 'Could not run curl-config' in e.message:
+    if ((hasattr(e, 'args') and
```

Comment:
a pattern more like this: http://stackoverflow.com/questions/11210104/check-if-a-program-exists-from-a-python-script